readme.md

```diff
@@ -246,14 +245,10 @@ gpg --armor --export-secret-keys %LONG_ID% | openssl base64
 *If you try to display the base64 encoded string in the terminal, some shells (like zsh or fish)
 may include an additional % character at the end, to mark the end of content which was not terminated by a newline character. This does not indicate a problem.
 Note for Windows - delete any linebreaks or spaces when copying the encoded string from terminal.*
-- `SONATYPE_PASSWORD`: The password you use to log into
-  https://s01.oss.sonatype.org/ (or https://oss.sonatype.org/ if your Sonatype
-  account was created before February 2021). Alternatively, the password part of
-  the user token if you generated one above.
-- `SONATYPE_USERNAME`: The username you use to log into
-  https://s01.oss.sonatype.org/ (or https://oss.sonatype.org/ if your Sonatype
-  account was created before 2021). Alternatively, the name part of the user
-  token if you generated one above.
+- `SONATYPE_PASSWORD`: The password part of your Sonatype
+  user token (not the account password!).
```

Comment:
should we add "generated on https://s01.oss.sonatype.org/ (or https://oss.sonatype.org/ if your Sonatype
  account was created before February 2021)." to clarify this is the Sonatype instance specific token not the Sonatype Portal one?

Reply:
Um, I don't know the difference. I will yield to any edit you suggest, but I'm not able to accurately make the edit myself.

Reply:
From 2024 onwards, we seem to have two tokens. The one described here I think is what's called OSSRH tokens in this announcement:

https://central.sonatype.org/news/20240617_migration_of_accounts/#the-impact-on-ossrh-and-the-central-publisher-portal
> While we have merged the account management process down to a single service, OSSRH publishing and Portal publishing are still different services. Your namespaces that are managed via OSSRH will continue to publish to the OSSRH server that you have previously used. Additionally, [OSSRH tokens](https://central.sonatype.org/publish/generate-token/) and [Portal tokens](https://central.sonatype.org/publish/generate-portal-token/) are different and currently cannot be used across services.